src/app/page.js

```diff
@@ -1,9 +1,135 @@
-import Image from 'next/image'
-
+"use client"
+import Image from 'next/image';
+import { useState } from "react";
+import { useRouter } from 'next/navigation';
 export default function Home() {
```

Comment:
**export default function(){}** can be used when the function doesn't have a name. There can only be one default export in a file. The alternative is a named export. This page describes export default in detail as well as other details about modules. Get to have a function name, so please check next js documentation it will save you a lot of stress.